src/Microsoft.Diagnostics.EventFlow.Inputs.ILogger/LoggerInputFactory.cs

```diff
@@ -0,0 +1,20 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Diagnostics.EventFlow.Inputs
+{
+    public class LoggerInputFactory : IPipelineItemFactory<LoggerInput>
+    {
+        public LoggerInput CreateItem(IConfiguration configuration, IHealthReporter healthReporter)
+        {
+            Validation.Requires.NotNull(configuration, nameof(configuration));
```

Comment:
Right now configuration is not used so I am not sure if checking for null is entirely warranted. But if you decide to implement level-based event filtering, you will need the configuration